src/index.js

```diff
@@ -63,12 +73,10 @@ program.action(function(cmd, cmd2) {
       "instead"
     );
   } else {
-    var suggestion = didYouMean(cmd, commandNames);
-    suggestion = suggestion || didYouMean([cmd, cmd2].join(":"), commandNames);
-    if (suggestion) {
-      logger.error();
-      logger.error("Did you mean", clc.bold(suggestion) + "?");
+    if (!didYouMean(cmd, commandNames)) {
+      cmd = [cmd, cmd2].join(":");
     }
+    suggestCommands(cmd, commandNames);
```

Comment:
if you refactor `suggestCommands` as mentioned above, you should be able to use that function twice here, once with the command combined and once without.

Actually, in doing that, you could solve the "keep 'Did you mean' output after `Error`" and simplifying the code a bit at the same time. Let me know if you need more direction 🙂